### DISCUSSION TITLE
Bookmark when Play/Pause is pressed on bluetooth or wired headphones.

### DISCUSSION THREAD

**Discussion by ErfolgreichCharismatisch** [PaulWoitaschek/Voice#1289] [Ideas]
**Is your feature request related to a problem? Please describe.**
I'm always frustrated when there is a great part in an audiobook and I have to take my phone out, fumble through the settings and set a bookmark.

**Describe the solution you'd like**
When I press the play/pause button on my bluetooth or wired headphones, a bookmark should be saved. I want the option to set a time earlier in seconds due to reaction time that is to always be used when doing that.

Then, to really play/pause, I manually enter the app.

**Describe alternatives you've considered**

- two quickly succeeding presses = Bookmark
- one press = play pause

**Comment by anaxonda** [comment]
I would love this feature as well. And have an intent for it so hardware buttons combinations could be remapped to it.
